js/components/main.js

```diff
@@ -86,7 +86,7 @@ class Main extends ImmutableComponent {
         location: url || Config.defaultUrl,
         isPrivate: !!options.isPrivate,
         isPartitioned: !!options.isPartitioned
-      })
+      }, options.openInForeground)
```

Comment:
is that just for added safety? WindowActions.newFrame defaults the argument to true (including if undefined)